SomosMasApp/SomosMasApp/App/Extensions/Extensions.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Extensions.swift
+//  SomosMasApp
+//
+//  Created by Mariano Battaglia on 17/03/2022.
+//
+
+import Foundation
+
+extension String {
+    
+    // Regular expressions to validate fields
+    var isValidEmail: Bool {
+        NSPredicate(format: "SELF MATCHES %@", "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}").evaluate(with: self)
+    }
+    
+    var isValidPassword: Bool {
+        NSPredicate(format: "SELF MATCHES %@", "^(?=.*[A-Za-z])(?=.*\\d)[A-Za-z\\d]{8,}$").evaluate(with: self)
+    }
+    
+    var isValidConfirmPassword: Bool {
```

Comment:
You don't need this and is against SOLID principles..... remember DRY(don't repeat yourself)